src/action/common/place_nix_configuration.rs

```diff
@@ -129,6 +129,10 @@ impl PlaceNixConfiguration {
             "extra-nix-path".to_string(),
             "nixpkgs=flake:nixpkgs".to_string(),
         );
+        settings.insert(
+            "upgrade-nix-store-path-url".to_string(),
+            "https://install.determinate.systems/nix-upgrade/stable/Universal".to_string(),
```

Comment:
We discussed either directly pointing at, or linking to, https://github.com/DeterminateSystems/nix-upgrade

Reply:
All set, thanks! Once the install.determinate.system update rolls out we'll be serving this by default:

```
$ curl -L https://install.determinate.systems/nix-upgrade/tag/v2.19.2+1/Universal
# Mirrored from https://releases.nixos.org/nix/nix-2.19.2/fallback-paths.nix
# via https://github.com/DeterminateSystems/nix-upgrade

{
  x86_64-linux = "/nix/store/smfmnz0ylx80wkbqbjibj7zcw4q668xp-nix-2.19.2";
  i686-linux = "/nix/store/knp0akbpj2k0rf26fmysmxdysmayihax-nix-2.19.2";
  aarch64-linux = "/nix/store/761hq0abn07nrydrf6mls61bscx2vz2i-nix-2.19.2";
  x86_64-darwin = "/nix/store/zlqvxis1dfcfgmy5fza4hllg6h03vhpb-nix-2.19.2";
  aarch64-darwin = "/nix/store/53r8ay20mygy2sifn7j2p8wjqlx2kxik-nix-2.19.2";
}
```

Reply:
👍 ready to roll :)